basculechecks/metricvalidator.go

```diff
@@ -177,3 +195,23 @@ func determineEndpointMetric(endpoints []*regexp.Regexp, urlHit string) string {
 	}
 	return "not_recognized"
 }
+
+func ProvideMetricValidator(server string) fx.Option {
+	return fx.Provide(
+		fx.Annotated{
+			Name: fmt.Sprintf("%s_bascule_capability_measures", server),
+			// TODO: this will be fixed when Metric Validator gets its own New()
```

Comment:
Will this be part of another PR?

Reply:
yeah: https://github.com/xmidt-org/bascule/issues/87 👍